R/semanticPage.R

```diff
@@ -23,24 +23,84 @@ get_dependencies <- function() {
   )
 }
 
+#' Get default semantic css
+#'
+#' @return path to default css semantic file
+get_default_semantic_theme <- function() {
+  # TODO cdf paths
+  if (getOption("shiny.minified", TRUE)) {
+    path <- system.file("semantic/semantic.min.css", package = "shiny.semantic")
+  } else {
+    path <- system.file("semantic/semantic.css", package = "shiny.semantic")
+  }
+  c(path)
+}
+
+#' Get default semantic js
+#'
+#' @return path to default js semantic file
+get_default_semantic_js <- function() {
+  # TODO cdf paths
+  if (getOption("shiny.minified", TRUE)) {
+    path <- system.file("semantic/semantic.min.js", package = "shiny.semantic")
+  } else {
+    path <- system.file("semantic/semantic.js", package = "shiny.semantic")
+  }
+  path
+}
+
 #' Semantic UI page
 #'
 #' This creates a Semantic page for use in a Shiny app.
 #'
 #' @param title A title to display in the browser's title bar.
+#' @param theme Theme name or path
 #' @param ... Other arguments to be added as attributes of the main div tag
 #' wrapper (e.g. style, class etc.)
 #'
 #' @export
-semanticPage <- function(..., title = "") { # nolint
+semanticPage <- function(..., title = "", theme = NULL) { # nolint
   content <- shiny::tags$div(class = "wrapper", ...)
 
   shiny::tagList(
-    get_dependencies(),
     shiny::tags$head(
+      shiny::tags$link(rel="stylesheet", href = check_semantic_theme(theme)),
```

Comment:
We want to by default load both dependencies (css & js) from our cdn, to achieve speedups. Currently, your default is going to load dependencies from Shiny which is slow. It is a good idea to add an option allowing to disable this behavior (maybe theme=FALSE?)